<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DependenciesUpgradeDialog</class>
 <widget class="QWidget" name="ReportDialog">
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QFormLayout" name="search_form">
        <property name="leftMargin">
         <number>10</number>
        </property>
        <property name="topMargin">
         <number>10</number>
        </property>
        <property name="rightMargin">
         <number>10</number>
        </property>
        <property name="bottomMargin">
         <number>10</number>
        </property>

        <item row="0" column="1">
        <layout class="QVBoxLayout" name="verticalLayout">
            
            <item row="2" >
              <layout class="QVBoxLayout" name="verticalLayout">

                <item>
                  <widget class="QLabel" name="report_label">
                    <property name="text">
                    <string> </string>
                    </property>
                  </widget>
                </item>
                <item>
                  <widget class="QTextEdit" name="report_widget">
                    <property name="text">
                    <string>Option: </string>
                    </property>
                  </widget>
                </item>

              </layout>
            </item>

        </layout>
        </item>

      </layout>
     </item>

    </layout>
   </item>

   
      
    <item row="1" column="0">
      <spacer name="horizontalSpacer">
        <property name="orientation">
        <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
        </property>
      </spacer>
    </item>

    <item row="2" column="0">
      <widget class="QWidget">
      
        <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
          <widget class="QPushButton" name="btn_upgrade">
          <property name="text">
            <string>Upgrade dependencies automatically</string>
          </property>
          </widget>
        </item>
        <item>
          <widget class="QPushButton" name="btn_cancel">
          <property name="text">
            <string>Cancel</string>
          </property>
          </widget>
        </item>
        </layout>

      </widget>
    </item>


   </layout>
  </widget>

 
 <resources/>
 <connections/>
</ui>
